library/src/main/java/com/airbnb/android/airmapview/AirMapView.java

```diff
@@ -80,6 +80,18 @@ public void initialize(FragmentManager fragmentManager, AirMapInterface mapInter
    * @param fragmentManager required for initialization
    */
   public void initialize(FragmentManager fragmentManager) {
+    initialize(fragmentManager, null, null);
+  }
+
+  /**
+   * Used for initialization of the underlying map provider. Uses the default preferred map provider
+   * (currently Native Google Maps, then Mapbox Web maps, then Web Google Maps).
+   *
+   * @param fragmentManager required for initialization
+   * @param mapboxAccessToken Mapbox Access Token (required for Mapbox maps)
+   * @param mapboxMapId Mapbox Map Id (required for Mapbox maps)
+   */
+  public void initialize(FragmentManager fragmentManager, String mapboxAccessToken, String mapboxMapId) {
```

Comment:
How about we require that the Mapbox token and app ID be included as [meta-data](http://developer.android.com/guide/topics/manifest/meta-data-element.html) in the `application` node of the `AndroidManifest.xml`. 

This aligns exactly with [the way Google Maps V2 works](https://developers.google.com/maps/documentation/android/signup) (see the section titled `Add the API key to your application`).